tests/indices/test_loading_graph.py

```diff
@@ -25,13 +25,27 @@ def test_load_graph_from_storage_simple(
         service_context=mock_service_context,
     )
 
+    # construct secon vector index
+    vector_index = GPTVectorStoreIndex.from_documents(
+        documents=documents,
+        storage_context=storage_context,
+        service_context=mock_service_context,
+    )
+
     # construct index
     list_index = GPTListIndex.from_documents(
         documents=documents,
         storage_context=storage_context,
         service_context=mock_service_context,
     )
 
+    # construct second list index
```

Comment:
shadowing the previously constructed `list_index` make it slightly harder to understand what this test is testing for.